scripts/run-pg-dev.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export PG_CONTAINER_NAME="postgres"
+PG_IMAGE_NAME="postgres:9.5"
+PG_HOST_DATA="/tmp/pgdata"
+PG_CONTAINER_DATA="/var/lib/postgresql/data"
+PG_USER="iroha"
+PG_PASS="mysecretpassword"
+export PG_PORT="5432"
+
+export G_ID=$(id -g)
+export U_ID=$(id -u)
+
+mkdir -p ${PG_HOST_DATA}
+docker run -dt --user "${U_ID}:${G_ID}" \
+        --name ${PG_CONTAINER_NAME} \
+        -p ${PG_PORT} \
```

Comment:
Consider replacing with `-p 127.0.0.1:5432:5432 \`, otherwise, there is a chance to bind Postgres to a public network interface (what was happen in my case).